README.md

```diff
@@ -25,22 +25,26 @@ _Eventually a bootstrap process will be handled by an [ado terraform provider](h
   - https://dev.azure.com/<your_organization>/osdu-r2/_git/osdu-r2
 
 - Create Empty Repositories (No Readme)
-  - osdu-infrastructure
   - entitlements-azure
   - legal
   - indexer-queue
   - storage
   - indexer
   - search
+  - delivery
 
-- Setup the Variable Group Neccessary for the Pipeline
+- Setup the Variable Group called 'Mirror Variables' necessary for the Pipeline
 
     | Variable | Value |
     |----------|-------|
     | ACCESS_TOKEN | <your_personal_access_token> |
-    | INFRASTRUCTURE_REPO | https://dev.azure.com/<your_organization>/osdu-r2/_git/osdu-infrastructure |
     | ENTITLEMENTS_REPO | https://dev.azure.com/<your_organization>/_git/entitlements-azure |
     | LEGAL_REPO | https://dev.azure.com/<your_organization>/osdu-r2/_git/legal |
+    | INDEXER_QUEUE_REPO | https://dev.azure.com/<your_organization>/osdu-r2/_git/indexer-queue |
```

Comment:
Also good except for delivery and when delivery is added then yes.

Reply:
Not sure what this means. Should I remove delivery from list?